tests/cluster/tests/28-cluster-shards.tcl

```diff
@@ -39,6 +42,17 @@ proc get_node_info_from_shard {id reference {type node}} {
     return {}
 }
 
+proc cluster_ensure_master {id} {
+    if {[R $id role] eq "master"} {
+        assert_equal {OK} [R $id CLUSTER FAILOVER]
```

Comment:
This is weird, if the role is master we issue `CLUSTER FAILOVER` to it? Should this this be if the role is slave

Reply:
Oops. This was originally `if { [regexp "master" [R $id role]] == 0 } {` which I suggesting changing to `eq` because it's simpler, but it should of course have been `ne`.

Reply:
Duh!